sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -562,6 +562,12 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ENABLE_REPLACE_CONDITIONAL_HASHJOIN =
+    conf("spark.rapids.sql.replaceConditionalHashJoin.enabled")
```

Comment:
indentation is a little off, but then again its off in other places in this file.